packages/web-components/src/components/va-accordion/va-accordion-item.tsx

```diff
@@ -35,28 +40,41 @@ export class VaAccordionItem {
    */
   @Prop() level: number = 2;
 
+  @State() slotHeader: string = null;
+
+  @State() slotTag: string = null;
+
   private toggleOpen(e: MouseEvent): void {
     this.accordionItemToggled.emit(e);
   }
 
+  private populateStateValues() {
+    getSlottedNodes(this.el, null).map(
```

Comment:
Since we don't care about the returned value, we could probably use `.forEach` instead of `.map`

Reply:
Updated to forEach, also with the `populateStateValues` function we are setting that on `<slot name="headline" onSlotchange={() => this.populateStateValues()}/>` so I don't think we'd have to worry about the filter because the element which nodes it's looping through is just that `<h*>` tag and not the parent where we might have a conflict with multiple slots during the loop